components/blitz/src/omero/util/IceMapper.java

```diff
@@ -422,6 +423,10 @@ public RType toRType(Object o) throws omero.ApiUsageException {
             Boolean b = (Boolean) o;
             omero.RBool bool = rbool(b.booleanValue());
             return bool;
+        } else if (o instanceof Class) {
+            Class c = (Class) o;
+            omero.RClass rc = rclass(c.getName());
```

Comment:
Didn't seem the right time to be modifying the class more significantly, so to keep the change here minimal, I copied the style of the rest of the checks.